File: Python/백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.py
import sys
input = sys.stdin.readline
l1,l2,l3 = map(int,input().split())

while l1 and l2 and l3:
    ll = [l1,l2,l3]
    if sum(ll) - max(ll) <= max(ll):
        print("Invalid")
    elif len(set(ll)) == 1:
        print("Equilateral")
    elif len(set(ll)) == 2:
        print("Isosceles")
    else:
        print("Scalene")
    l1,l2,l3 = map(int,input().split())

File: C++17/백준/Bronze/5073. 삼각형과 세 변/삼각형과 세 변.cc
#include <iostream>

int main(void){
    int a,b,c;
    
    std::cin >> a >> b >> c;
    
    while(a + b + c > 0){
        int sum = a+b+c;
        
        if (a == b && b == c){
            std::cout << "Equilateral" << std::endl;
        }
        else if (sum <= 2 * std::max(std::max(a,b),c)){
            std::cout << "Invalid" << std::endl;
        }
        else if (a != b && b != c && a != c){
            std::cout << "Scalene" << std::endl;
        }
        else{
            std::cout << "Isosceles" << std::endl;
        }
        
        std::cin >> a >> b >> c;
    }
    
    return 0;
}